Testscripts/Linux/SetupRDMA.sh

```diff
@@ -390,9 +390,18 @@ function Main() {
 	cd mpi-benchmarks/src_c
 	LogMsg "Building Intel MPI Benchmarks tests"
 	make -j $(nproc)
-	LogMsg "Completed Intel MPI Benchmarks"
 	Verify_Result
-	LogMsg "Intel Benchmark test installation completed"
+
+	# install P2P test
+	LogMsg "Change directory to P2P"
+	cd P2P
+	LogMsg "Renaming from mpiicc to mpicc in Makefile"
+	sed -i -e 's/CC=mpiicc/CC=mpicc/g' Makefile
+	LogMsg "Building P2P binary"
+	make -j $(nproc)
+	LogMsg "Completed P2P2 binary compilation"
```

Comment:
P2P2 or P2P?

Reply:
fixed in 9c875c4e590774f8a8505f5b9e1f86d9236e4be4